pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -311,7 +311,7 @@ stdenv.mkDerivation (finalAttrs: {
     ] ++ lib.optionals (mysqlSupport) [
       "-L" "${libmysqlclient}/lib"
       "-I" "${libmysqlclient}/include"
-    ] ++ lib.optional (qttranslations != null) [
+    ] ++ lib.optional (withQttranslation && (qttranslations != null)) [
```

Comment:
On what is this based? It does not seem to depend on gtk3 directly.

Reply:
When x11 support is disabled, qtbase stops building - `qt translation` requires x11 support.

Reply:
```suggestion
    ] ++ lib.optional withQttranslation [
    # depends on x11
```

Reply:
Error with this variant:
```
       error: cannot coerce null to a string

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-5/pkgs/development/libraries/qt-5/modules/qtbase.nix:316:26:

          315|       # depends on x11
          316|       "-translationdir" "${qttranslations}/translations"
             |                          ^
          317|     ]
```

Reply:
Returned previous variant.